index.html

```diff
@@ -123,6 +123,7 @@
 									<li class="hidden-sm hidden-xs"><a href="http://blog.fossasia.org/contact/" target="_self">Contact</a></li>											
 									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
 									<li class="social-link hidden-md hidden-sm hidden-xs"><a target="_self" href="https://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
+<li><a target="_self" href="https://github.com/fossasia"><i class="fa fa-github fa-lg"></i></a></li>
```

Comment:
Please add `class="social-link hidden-md hidden-sm hidden-xs"` to `li` here that will solve the icon alignment issue.